### PR TITLE
improve preface

### PR DESCRIPTION
**Pull request by michaeldorman**

### CODE REVIEW

**Review by Nowosad — APPROVED** [review]

File: preface.qmd [hunk 1]
```diff
@@ -7,7 +7,7 @@ We aim to *minimize surprises*, with each section and chapter building on the pr
 If you're just starting out with Python for working with geographic data, this book is an excellent place to start.
 
 There are many resources on Python on 'GeoPython' but none that fill this need for an introductory resource that provides strong foundations for future work.
-We want to avoid reinventing the wheel and provide something that fills an 'ecological niche' in the wider free and open source software for geospatial (FOSS4G) ecosystem.
+We want to avoid reinventing the wheel and provide something that fills an 'ecological niche' in the wider free and open-source software for geospatial (FOSS4G) ecosystem.
```
Comment (Robinlovelace) [inline preface.qmd:10]: Was going to question this but, having looked it up, I think this is right and good, in open-source software it's an adjective and that seems right: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source

File: preface.qmd [hunk 2]
```diff
@@ -16,12 +16,12 @@ Key features include:
 <!-- 4.  Exercises at the end of each chapter with reproducible and open solutions -->
 1.  Provision of lucid example datasets and meaningful operations to illustrate the applied nature of geographic research
 
-This book is complementary with, and adds value to, other projects in the ecosystem, as highlighted in the following comparison between *Geocomputation with Python* and related GeoPython books:
+This book is complementary, and adds value to, other projects in the ecosystem, as highlighted in the following comparison between *Geocomputation with Python* and related GeoPython books:
```
Comment (Robinlovelace) [inline preface.qmd:19]:
```suggestion
This book complements and adds value to other projects in the ecosystem, as highlighted in the following comparison between *Geocomputation with Python* and related GeoPython books:
```

shorter and simpler, right?

**Review by Robinlovelace — APPROVED** [review]

File: preface.qmd (hunk 2 above)
Comment (michaeldorman) [reply preface.qmd]: Right, thanks!

**Comment by Robinlovelace** [comment]
Two :+1: is enough for now so will merge, great work Michael!

**Pull request merged**